docs/config/services/deployments/production.js

```diff
@@ -7,15 +7,15 @@ var angularCodeUrl = '//code.angularjs.org/';
 
 var cdnUrl = googleCdnUrl + versionInfo.cdnVersion;
 
-// The plnkr examples must use the code.angularjs.org repo for the snapshot,
-// and the cdn for the tagged version and, if the build is not tagged, the currentVersion.
-//
-// The currentVersion may not be available on the cdn (e.g. if built locally, or hasn't been pushed
-// yet). This will lead to a 404, but this is preferable to loading a version with which the example
-// might not work (possibly in subtle ways).
+// The "examplesCdnUrl" here applies to the examples when they are opened in plnkr.co.
+// The embedded examples instead always include the files from the *default* deployment,
+// to ensure that the source files are always available.
+// The plnkr examples must always use the code.angularjs.org source files.
+// We cannot rely on the CDN files here, because they are not deployed during the same time
+// as the docs app.
 var examplesCdnUrl = versionInfo.currentVersion.isSnapshot ?
```

Comment:
Since it is no longer an actual CDN let's change this variable to something like `examplesPath`.